react/react1/week1/app/src/components/TodoList.jsx

```diff
@@ -0,0 +1,13 @@
+import Todo from "./Todo";
+
+function TodoList({ todos }) {
+  return (
+    <ol>
+      {todos.map((todo) => (
+        <Todo description={todo.description} deadline={todo.deadline} />
```

Comment:
You could pass `todo={todo}`, and access the props inside the `Todo.jsx` component, since you're not really using these props here on `TodoList.jsx`
Also, your console has a warning, because each child has to have a key. :)

Reply:
Hi Anna,
It was really nice of you to review my hw.
Thanks for all the suggestions.